androidshared/src/main/java/org/odk/collect/androidshared/ui/MultiSelectViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package org.odk.collect.androidshared.ui
+
+import androidx.lifecycle.ViewModel
+import org.odk.collect.androidshared.livedata.MutableNonNullLiveData
+import org.odk.collect.androidshared.livedata.NonNullLiveData
+
+class MultiSelectViewModel : ViewModel() {
```

Comment:
We have several places in the app that use multi select lists that we want to reimplement, and this should allow us to do that without duplicating code for the actual selection logic (and the persistence of the selection state between configuration changes).